### ISSUE TITLE
Avoid multiple protein definitions in PPI map view

### ISSUE THREAD

**Issue by sacdallago** [sacdallago/cellmap#19]
If I send a link with the same protein twice (eg. Duplicates in the link), then the map renders two different look ts for the same protein and it messes up the visualization / PPI network
Related to #18

**Comment by sacdallago** [comment]
Hypothesis: The `if` clause checking if the protein is already in the view will compile to false because the protein get's added via callback after an ajax request (thus, there is some time difference between the when added to the view and the check to add it again).

Might be solved by implementing #21

**Comment by danielgomz** [comment]
So we are looking for a replacement for a several lines that are occurring during the return of the uniprotId?

**Comment by sacdallago** [comment]
Hi @danielgomz :) Sorry, I'm a bit out of this, was using this issue as a note-to-self. Could you tell me in greater detail what you (as in your _we_) are interested in? :)

**Comment by danielgomz** [comment]
Yeah, I was curious about how you perform protein prediction ? My interest lies in the precision and accuracy. I would also like to know if there is any binding site ligand interaction interfaces that can be implemented.

**Comment by sacdallago** [comment]
@danielgomz I don't perform any predictions on cellmap? I may be lost...

**Comment by danielgomz** [comment]
can you review the purpose of cell?
